pkg/events/parse_args.go

```diff
@@ -198,17 +198,51 @@ func ParseArgs(event *trace.Event) error {
 				ParseOrEmptyString(modeArg, inodeModeArgument, err)
 			}
 		}
-	case SecuritySocketSetsockopt, Setsockopt, Getsockopt:
+	case SecuritySocketSetsockopt, Setsockopt:
 		if levelArg := GetArg(event, "level"); levelArg != nil {
 			if level, isInt := levelArg.Value.(int32); isInt {
 				levelArgument, err := helpers.ParseSocketLevel(uint64(level))
 				ParseOrEmptyString(levelArg, levelArgument, err)
 			}
 		}
+		customOptionsStringsMap := map[uint64]string{
+			helpers.SO_ATTACH_OR_GET_FILTER.Value(): "SO_ATTACH_FILTER",
+			helpers.SO_DETACH_FILTER_OR_BPF.Value(): "SO_DETACH_FILTER",
```

Comment:
This is the search of the `SO_DETACH_BPF` symbol in the kernel:
![image](https://user-images.githubusercontent.com/87267148/189602577-a2f849a5-3f98-4aeb-ab86-797a259a0075.png)
You can see that there is no place that use this constant.
Practically the both of them are used as `SO_DETACH_FILTER` because filter is a BPF program.
In the attachment the difference is between BPF and eBPF in the filter.